.changeset/lucky-rivers-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@directus/sdk': patch
```

Comment:
Since we're changing the signature of the shared library websocket interface which can impact compatibility with 3rd party websocket libraries this should be a minor or perhaps even a major change.

Reply:
What's the incompatibility? 🤔
This works fine:
<img width="880" src="https://github.com/directus/directus/assets/5363448/1033311c-8238-4304-a495-a20799a81b4d">

Reply:
That is just one of many websocket implementations, it can potentially impact 3rd party implementations that rely on our current `URL` based shared interface.

Reply:
That seems very unlikely to me 🤔 But we can mark it as a breaking change if you prefer so 👍

Reply:
If anything, it should be a major update then. Going to update the changeset 👍

Reply:
But probably not worth adding an entry to the breaking change page? WDYT?

Reply:
I think that may indeed be overkill, as you said "That seems very unlikely to me". I doubt anyone is actually relying on this currently (and if someone does run into it it should be a really easy fix)